ONSdigital/ssdc-rm-docker-dev
# Rename variable to iap-enabled

# Motivation and Context

We want to make it more obvious that the support tool is running under IAP without the dummy user present.
# What has changed



Variable renamed to IAP_ENABLED
# How to test?



Build the [support tool branch](https://github.com/ONSdigital/ssdc-rm-support-tool/pull/322) and do a make up, run the ATs and it should run with the dummy user present.
# Links



https://jira.ons.gov.uk/browse/SDCSRM-289
# Screenshots (if appropriate):
